scripts/build/macOS/NotarizeMacos.cmake

```diff
@@ -0,0 +1,111 @@
+# CMake script to sign macOS build
+# Arguments:
+# APP_IDENTIFIER - app identifier
+# APP_LOCATION - the path to Audacity.app
+# DMG_LOCATION - the path to Audaicty dmg package
+# APPLE_NOTARIZATION_USER_NAME - notarization user name
+# APPLE_NOTARIZATION_PASSWORD - notarization password
+
+# https://cmake.org/cmake/help/latest/policy/CMP0054.html
+cmake_policy( SET CMP0054 NEW )
+# https://cmake.org/cmake/help/latest/policy/CMP0011.html
+cmake_policy( SET CMP0011 NEW )
+
+function( get_plist_value output path key )
+    execute_process(
+        COMMAND /usr/libexec/PlistBuddy -c "Print ${key}" "${path}"
+        OUTPUT_VARIABLE result
+    )
```

Comment:
You could add `RESULT_VARIABLE exit_status` to [`execute_process()`](https://cmake.org/cmake/help/v3.15/command/execute_process.html), then afterwards:

```cmake
if(NOT "${exit_status}" EQUAL "0")
    message(FATAL_ERROR "PlistBuddy command failed with status ${exit_status}. See output above for details.")
endif()
```
Same for other occurrences of `execute_process()`.

Reply:
I've checked this - cmake will fail if execute_process failed with a verbose error